### PR TITLE
Chore(docs): Release notes 2023.01

### LINKED ISSUE #8610: Chore: Release notes 2023.01
Release notes 2023.01

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #8610

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -3,6 +3,124 @@ id: Release notes
...
+### Log viewer:**
+- Removed log viewer as it has it's own repo ([#8571](https://github.com/patternfly/patternfly-react/pull/8571))
+
+### Table
+- Added menuAppendTo props to actions tables ([#8521](https://github.com/patternfly/patternfly-react/pull/8521))
+- Fixed verbiage/formatting in example documentation ([#8565](https://github.com/patternfly/patternfly-react/pull/8565))
+- Fixed draggable column management demo ([#8553](https://github.com/patternfly/patternfly-react/pull/8553))
+- Fixed parent checkbox behavior in composable table ([#8490](https://github.com/patternfly/patternfly-react/pull/8490))
+
+### Topology
```
Comment (jenny-s51) [inline packages/react-docs/RELEASE-NOTES.md:105]: 🤩

**Review by jenny-s51 — APPROVED** [review]
Looking awesome!

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -3,6 +3,124 @@ id: Release notes
...
+  - Recreated all legacy wizard examples ([#8422](https://github.com/patternfly/patternfly-react/pull/8422))
+
+### Access console extension
+- Introduced RFB clipViewport and dragViewport options ([#8470](https://github.com/patternfly/patternfly-react/pull/8470))
+- Removed react console extension from patternfly-react repo ([#8575](https://github.com/patternfly/patternfly-react/pull/8575))
+
+### Catalog view extension
+- Removed catalog view extension from patternfly-react repo ([#8474](https://github.com/patternfly/patternfly-react/pull/8474))
+
+### Log viewer:**
```
Comment (jenny-s51) [inline packages/react-docs/RELEASE-NOTES.md:96]:
```suggestion
### Log viewer:
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -3,6 +3,124 @@ id: Release notes
...
+- **Expandable section:**
+  - Made toggle button conditional with truncate variant ([#8570](https://github.com/patternfly/patternfly-react/pull/8570))
+- **File upload:**
+  - Updated `dropzoneProps` to be partial ([#8407](https://github.com/patternfly/patternfly-react/pull/8407))
+- **Label:**
+  - Added event parameter to beta callback props ([#8544](https://github.com/patternfly/patternfly-react/pull/8544))
+- **Label group:**
+  - Updated to show the correct count of overflow labels ([#8505](https://github.com/patternfly/patternfly-react/pull/8505))
+- **Menu:**
+  - Added event param to `onDrillIn` and `onDrillOut` beta callbacks ([#8514](https://github.com/patternfly/
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:46]: Missing closing parenthesis are breaking the displayed link

```suggestion
  - Added event param to `onDrillIn` and `onDrillOut` beta callbacks ([#8514](https://github.com/patternfly/))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -3,6 +3,124 @@ id: Release notes
...
+- **Search input:**
+  - Edited `ExpandableInput` and `SearchAttribute` for consistency ([#8527](https://github.com/patternfly/patternfly-react/pull/8527))
+  - Reordered props in onChange and onSearch ([#8516](https://github.com/patternfly/patternfly-react/pull/8516))
+  - Added a name prop to the input ([#8554](https://github.com/patternfly/patternfly-react/pull/8554))
+- **Select:**
+  - Added toggle ref ([#8528](https://github.com/patternfly/patternfly-react/pull/8528))
+- **Select next:**
+  - Updated logic for keyboard interaction ([#8496](https://github.com/patternfly/patternfly-react/pull/8496))
+- **Simple list:**
+  - Added aria label added to ungrouped simple list ([#8507](https://github.com/patternfly/patternfly-react/pull/8507))
```
Comment (evwilkin) [inline packages/react-docs/RELEASE-NOTES.md:67]: Duplicate "added"

```suggestion
  - Added aria label to ungrouped simple list ([#8507](https://github.com/patternfly/patternfly-react/pull/8507))
```

**Review by jenny-s51 — COMMENTED** [review]
For consistency and clarity, it would be nice to include "docs" in the verbiage for all of these bullets!

**Review by evwilkin — APPROVED** [review]
LGTM

File: packages/react-docs/RELEASE-NOTES.md (hunk 2 above)
Comment (jenny-s51) [reply packages/react-docs/RELEASE-NOTES.md:113]: Looks like we still want to push this small change! Otherwise release notes look great 🙂

**Review by nicolethoen — APPROVED** [review]

**Pull request merged**